src/WebUI/locales/en.yml

```diff
@@ -745,15 +745,15 @@ user:
     gold: Gold
     heirloom: Heirlooms points
   settings:
-    title: Account settings
+    title: Account Deletion
     delete:
```

Comment:
Settings is settings. Not Deletion. This category is meant to have more settings later

Reply:
I was working on premise of that it could be changed back once/if more settings are available, but I am fine to change back - Will fix now for all localisations and resolve once done

Reply:
Updated and reverted, marking as resolved.